apps/dashboard/app/models/project.rb

```diff
@@ -17,10 +18,13 @@ def all
     end
 
     def find(project_path)
+      # previously the @proj_name kicked off a whole chain of events firing
+      # now, we are just getting that manifest entry back only
+      #
       full_path = dataroot.join(project_path)
       return nil unless full_path.directory?
 
-      Project.new({ name: full_path.basename })
+      Project.new( name: full_path.basename )
```

Comment:
This currently breaks at the update as there is only a `name` to be served back to populate fields.